src/main/webapp/index2.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Team 23 CodeU Locator</title>
+  <script async defer src="https://maps.googleapis.com/maps/api/js?key=AIzaSyCdoTmSnZmgvvWPSBtOd79Ae20aYhfMGN8&callback=initMap"
+  type="text/javascript"></script>
+  <script>
+    function createMap(){
+      const map = new google.maps.Map(document.getElementById('map'), {
+        center: {lat: 37.422, lng: -122.084},
+        zoom: 16
+      });
+      const trexMarker = new google.maps.Marker({
+        position: {lat: 37.421903, lng: -122.084674},
+        map: map,
+        title: 'Stan the T-Rex'
+      });
+
+
+      var trexInfoWindow = new google.maps.InfoWindow({
+      content: 'This is where our team is located!'
+      });
```

Comment:
On line 21, you should indent it by 2 more spaces to keep the indenting format consistent with the rest of the document. Either that or you can also just put lines 22 and 21 all at the end of line 20.